### DISCUSSION TITLE
certificate expired when trying to verify signatures after downloading keepassxc

### DISCUSSION THREAD

**Discussion by borg-cube** [keepassxreboot/keepassxc#7150] [Q&A] [answered]
Dear developers, users,

when trying to fetch gpg key durig downloading the tool the following error occurs (i think the german ist not a problem):
> gpg --fetch-keys https://keepassxc.org/keepassxc_master_signing_key.asc
gpg: fordere Schlüssel von https://keepassxc.org/keepassxc_master_signing_key.asc an
gpg: WARNUNG: die URI https://keepassxc.org/keepassxc_master_signing_key.asc kann nicht geholt werden: Zertifikat abgelaufen

> gpg --keyserver keys.openpgp.org --recv-keys CFB4C2166397D0D2
gpg: Empfangen vom Schlüsselserver fehlgeschlagen: Zertifikat abgelaufen

the certificate is expired. 

I wanted to report this. maybe you can give a hint or update?

Thanks in advance!

I used the hints on page https://keepassxc.org/verifying-signatures/

**Comment by droidmonkey** [comment, marked as answer]
`gpg --refresh-keys`

**Reply by borg-cube** [reply]
was not successful. fetching keys still throws the error but when verifying I got the **expected result:**

gpg: die unterzeichneten Daten sind wohl in 'KeePassXC-2.6.6-Win64.msi'
gpg: Signatur vom 12.06.2021 04:24:06 Mitteleuropische Sommerzeit
(...)
**gpg: Korrekte Signatur von "KeePassXC Release <release@keepassxc.org>" [unbeka**
gpg: WARNUNG: Dieser Schlüssel trägt keine vertrauenswürdige Signatur!
gpg:          Es gibt keinen Hinweis, daß die Signatur wirklich dem vorgeblich
Haupt-Fingerabdruck  = BF5A 669F 2272 CF43 24C1  FDA8 CFB4 C216 6397 D0D2
Unter-Fingerabdruck  = C1E4 CBA3 AD78 D3AF D894  F9E0 B7A6 6F03 B590 76A8

Thanks for yout time, bye

**Reply by phoerious** [reply]
That is a warning that you haven't trusted the key. The signature itself is valid.
